sdk/cognitivelanguage/ai-language-text/test/public/analyze.spec.ts

```diff
@@ -255,6 +256,20 @@ matrix([["APIKey", "AAD"]] as const, async (authMethod: AuthMethod) => {
           );
         });
 
+        it("service returns script with DetectLanguageInput[]", async function () {
+          const doc = ["Tumhara naam kya hai?"];
+          const docs = doc.map((text) => ({
+            id: getId(),
+            text,
+          }));
+          assertActionResults(
+            await client.analyze(AnalyzeActionNames.LanguageDetection, docs, {
+              modelVersion: "2022-04-10-preview",
+            }),
+            expectation72
+          );
```

Comment:
nitpick: could it be simplified to the following?
```suggestion
          const docs = ["Tumhara naam kya hai?"];
          assertActionResults(
            await client.analyze(AnalyzeActionNames.LanguageDetection, docs, "en", {
              modelVersion: "2022-04-10-preview",
            }),
            expectation72
          );
```